homeassistant/components/fully_kiosk/button.py

```diff
@@ -39,31 +39,31 @@ class FullyButtonEntityDescription(
 BUTTONS: tuple[FullyButtonEntityDescription, ...] = (
     FullyButtonEntityDescription(
         key="restartApp",
-        name="Restart browser",
+        translation_key="restart_browser",
         device_class=ButtonDeviceClass.RESTART,
         entity_category=EntityCategory.CONFIG,
         press_action=lambda fully: fully.restartApp(),
     ),
     FullyButtonEntityDescription(
         key="rebootDevice",
-        name="Reboot device",
+        translation_key="",
```

Comment:
This is an error I guess, however, considering we have two buttons (restart browser & restart device), I think adding a translation string is needed to prevent confusion.

../Frenck